Samples/StyleTransfer/App.xaml.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
```

Comment:
This should be Microsoft.UI.* right?

Reply:
Yeah... interesting. It seems like the WinUI Microsoft.* is missing MediaElement, and MediaPlayerElement per: https://docs.microsoft.com/en-us/windows/apps/winui/winui3/#preview-1-limitations-and-known-issues.